### PR TITLE
Day7 work

### PR DESCRIPTION
**Pull request by frunzisss**

### CODE REVIEW

File: js/View/new/newViews.js [hunk 1]
```diff
@@ -1,373 +1,137 @@
...
+    <div class="form-section">
+        <span class="form-section-span formatted primaryColor">${options.label}</span></br>
+        <textarea class="textarea textarea-skin" name="${options.name}" placeholder="${options.placeholder}"></textarea>
+    </div>
+    `;
+};
+
+const Form = (options) => {
+    var textAreas = options.data.map((el) => {
+        return Textarea(el);
```
Comment (gcazaciuc) [inline js/View/new/newViews.js:24]: you can do it without {} and return

File: js/Models/data.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
+{
```
Comment (gcazaciuc) [inline js/Models/data.js:1]: pls put these files into src/data folder. models we will reserve for real model classes not for raw test data

**Review by gcazaciuc — APPROVED** [review]

File: js/Models/data.js (hunk 2 above)
Comment (frunzisss) [reply js/Models/data.js:1]: sure :)

File: js/View/new/newViews.js (hunk 1 above)
Comment (frunzisss) [reply js/View/new/newViews.js:24]: options.data.map(el => Textarea(el));

**Pull request merged**